helix-core/src/main/java/org/apache/helix/task/TaskStateModel.java

```diff
@@ -283,6 +292,38 @@ public void reset() {
     }
   }
 
+  /**
+   * Loads Task and TaskFactory classes for command input
+   */
+  private boolean loadNewTask(String command) {
```

Comment:
Is there a better way to design this method signature - instead of returning a boolean?